src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -47,6 +48,10 @@ public class ParameterProvider {
   public static final String ENABLE_NEW_JSON_PARSING_LOGIC =
       "ENABLE_NEW_JSON_PARSING_LOGIC".toLowerCase();
 
+  public static final String PARQUET_WRITER_VERSION = "PARQUET_WRITER_VERSION".toLowerCase();
+  public static final String ENABLE_PARQUET_DICTIONARY_ENCODING =
+      "ENABLE_PARQUET_DICTIONARY_ENCODING".toLowerCase();
```

Comment:
leaving it here so we don't forget it : PARQUET_WRITER_VERSION will be dynamic depending on the openchannelresponse

Slso don't make ENABLE_PARQUET_DICTIONARY_ENCODING into a public member. We don't want customers to be exposed to it just yet.

Reply:
Changed to dynamic parquet writer version. Removed `ENABLE_PARQUET_DICTIONARY_ENCODING` , always enable dict encoding in iceberg mode.